sdk/containerregistry/Azure.Containers.ContainerRegistry/tests/ContainerRepositoryClientLiveTests.cs

```diff
@@ -109,7 +113,256 @@ public async Task CanSetRepositoryProperties()
             // Cleanup
             await client.SetPropertiesAsync(originalContentProperties);
         }
+        #endregion
 
+        #region Registry Artifact Tests
+        [RecordedTest]
+        public async Task CanGetMultiArchitectureImageProperties()
+        {
+            // Arrange
+            ContainerRepositoryClient client = CreateClient();
+            string tag = "v1";
+            int helloWorldManifestReferences = 9;
+
+            // Act
+            RegistryArtifactProperties properties = await client.GetRegistryArtifactPropertiesAsync(tag);
+
+            // Assert
+            Assert.Contains(tag, properties.Tags.ToList());
+            Assert.AreEqual(_repositoryName, properties.Repository);
+            Assert.GreaterOrEqual(helloWorldManifestReferences, properties.RegistryArtifacts.Count);
+
+            Assert.IsTrue(properties.RegistryArtifacts.Any(
+                artifact => {
+                    return artifact.CpuArchitecture == "arm64" &&
+                           artifact.OperatingSystem == "linux"; } ));
```

Comment:
super nit: this can be simplified slightly to:
```suggestion
                artifact => 
                    artifact.CpuArchitecture == "arm64" &&
                    artifact.OperatingSystem == "linux" ));
```